src/util.h

```diff
@@ -31,7 +31,7 @@
 
 /** @brief Execute a shell command
  */
-int execute(const char cmd_line[], int quiet);
+int execute(const char cmd_line[], int quiet, char *msg, size_t msg_len);
```

Comment:
It might be nicer if we add default arguments to msg and msg_len. That way you do not need to add .. NULL, 0) to many execute calls.